plugins/available/git.plugin.bash

```diff
@@ -1,275 +1,314 @@
+# shellcheck shell=bash
+
 cite about-plugin
 about-plugin 'git helper functions'
 
 function git_remote {
-  about 'adds remote $GIT_HOSTING:$1 to current repo'
-  group 'git'
+	about "adds remote $GIT_HOSTING:$1 to current repo"
```

Comment:
Interesting that this `about` was ever used. I don't think it would work as intended when looking at `glossary`, but I haven't tested.